doc/classes/AnimationNodeStateMachine.xml

```diff
@@ -52,6 +52,12 @@
 				Returns the animation node with the given name.
 			</description>
 		</method>
+		<method name="get_node_list" qualifiers="const">
+			<return type="PackedStringArray" />
+			<description>
+				Returns the names of all nodes in the sub-tree.
```

Comment:
Ditto.
```suggestion
				Returns the names of all nodes in this tree.
```

Reply:
It's what the class description calls it. "A sub-tree of many type AnimationNodes used for complex animations. Used by AnimationTree."